plugins/org.jkiss.dbeaver.ui.editors.sql/src/org/jkiss/dbeaver/ui/editors/sql/internal/SQLEditorMessages_ro.properties

```diff
@@ -193,8 +193,6 @@ pref_page_sql_format_label_format_active_query_tip = Formateaz\u0103 numai inter
 pref_page_sql_completion_group_sql_assistant = Asistent SQL/finalizare
 pref_page_sql_completion_label_enable_auto_activation = Activa\u0163i activarea automat\u0103
 pref_page_sql_completion_label_enable_auto_activation_tip = Activeaz\u0103 activarea automat\u0103 a Asistentului de con\u0163inut (la tastarea textului)
-pref_page_sql_completion_label_enable_experimental_features = Activa\u0163i func\u0163iile experimentale
```

Comment:
Do we really want to remove such localization? As I remember @serge-rider usually ask to translate it and update rather than delete it

Reply:
It's In progress